src/main/scala/com/gu/support/workers/model/monthlyContributions/state/CreatePaymentMethodState.scala

```diff
@@ -2,12 +2,17 @@ package com.gu.support.workers.model.monthlyContributions.state
 
 import java.util.UUID
 
+import com.gu.acquisition.model.{OphanIds, ReferrerAcquisitionData}
 import com.gu.support.workers.model.monthlyContributions.Contribution
 import com.gu.support.workers.model.{PayPalPaymentFields, StripePaymentFields, User}
+import ophan.thrift.event.AbTest
 
 case class CreatePaymentMethodState(
   requestId: UUID,
   user: User,
   contribution: Contribution,
-  paymentFields: Either[StripePaymentFields, PayPalPaymentFields]
+  paymentFields: Either[StripePaymentFields, PayPalPaymentFields],
+  ophanIds: OphanIds,
```

Comment:
If you were to put all the new fields into a containing class and have this class have an Option of it it might make things easier? I don't mind it this way but it's going to be quite a bit of work to update all the test fixtures I would think.

This obviously applies to all the other places that these fields have been added

Reply:
Ok - will do. Although does it have to be optional?

Reply:
Not necessarily although that would make it easier to work with in tests and also you could deploy it ahead of the support-frontend changes which pass these things through. Up to you though, if you're not bothered about updating all the fixtures and coordinating the release a bit more carefully then I'm fine with it not being an option.

Reply:
Ok. Had a chat with @desbo. We like the idea of deploying ahead of the support-frontend changes, as we can then start to monitor for acquisition events being sent, even without the work on support-frontend being merged. Once the work in support-frontend has been merged, we'll then review to see if we think its worth the effort changing the field to a required one.